### ISSUE TITLE
Show fasets / Show count link missing

### ISSUE THREAD

**Issue by gleniat** [unfoldadmin/django-unfold#700]
unfold 0.36.0
django 5.1
latest Chrome

**Did you searched other issues, if the bug is not already fixed?**

Yes

**Describe your issue**
Django 5 introduced "Show count" link in filter box.
https://docs.djangoproject.com/en/5.1/ref/contrib/admin/#django.contrib.admin.ModelAdmin.show_facets

```
<a href="?_facets=True" class="viewlink">Show count</a>
<a href="?" class="hidelink">Hide count</a>
```

The link is missing in Unfold theme.

![image](https://github.com/user-attachments/assets/173461ac-cb75-42c4-b780-b940d6830452)

![image](https://github.com/user-attachments/assets/8c6e2388-d99f-47c9-83e1-0e79ee3cf8df)

When I manually append `?_facets=True` to the URL, the counts display correctly. The only missing piece is the link.

![image](https://github.com/user-attachments/assets/31227dbb-8172-46e2-85c7-90db6da63224)

**Comment by lukasvinclav** [comment]
Thanks for report.

**Comment by lukasvinclav** [comment]
Fixed in #717
